hack/patch.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -xe
+
+source ./hack/common.sh
+
+kubectl=./cluster-up/kubectl.sh
```

Comment:
Consider doing the following imports:

```bash
source hack/common.sh
source cluster-up/cluster/$KUBEVIRT_PROVIDER/provider.sh
source hack/config.sh
```

then you have `_kubectl` available which points to the right locations (e.g. different for external provider).

Reply:
Done